index.html

```diff
@@ -267,18 +265,18 @@ <h1>
                     </li>
                     <li>
                       <a href="https://github.com/mbonabucya">
-                        <img src="image/Vector-peace.svg" alt=" peace icon" />
+                        <img src="image/Vector-peace.svg" alt="peace icon" />
                       </a>
                   </li>
                 </ul>
-                <button class="resume">
+                <button class="resume" type="submit">
                     Get my resume
                 </button>
             </div>
             <div class="skills">
                 <div class="skills-header">
                     <h1>Languages</h1> 
-                     <img src="image/dropdown.png" alt=" drop down icon" />
+                     <img src="image/dropdown.png" alt="drop-down icon" />
                 </div>
                 <div class="language-skills">
                     <ul>
```

Comment:
The alternative text for this section is not particularly descriptive. Please change your alternative text to match the technologies listed in this section. Make sure you type them correctly as well, respecting capitalization (HTML, CSS, and JavaScript).

Additionally, the placeholder text for your inputs makes for bad UI/UX. Please change them to something like `Enter name here` and `Enter email here`.